hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+                line=dict(color=Q7, width=3),
+                name ='Linear Regression Model'
+                )
+
+    layout = go.Layout(xaxis=dict(ticks='', showticklabels=True,
+                              zeroline=False, title=Q4),
+                   yaxis=dict(ticks='', showticklabels=True,
+                              zeroline=False, title=Q5),
+                   showlegend=True, hovermode='closest',
+                   title=Q2
+                   )
+
+    fig = go.Figure(data=[p1, p2], layout=layout)
+
+    return py.iplot(fig)
+
+
+def question(data2,X_train,X_test,y_train,y_test, Q1):
+    if Q1 == '2':
+        x1 = input("Please enter X variable:")
+        x = data2[x1]
+        x2 = input("Please enter X2 variable (for comparison):")
+        xx = data2[x2]
+        y = input("Please enter Y variable:")
+        x_lab = input("Please enter the label for the X-Axis:")
+        y_lab = input("Please enter the label for the Y-Axis:")
+        title = input("Please enter the title of the graph:")
+        return scatter(x,xx, y, x_lab, y_lab, title,data2);
+
+    if Q1 == '1':
+        x1 = input("Please enter a variable:")
+        x = data2[x1]
+        x_lab = input("Please enter the label for the X-Axis:")
+        title = input("Please enter the title of your graph:")
+        return count_and_print(x, x_lab, title);
+
+
+    if Q1== '3':
+        x = data2[input("Please enter X variable name:")]
+        y = data2[input("Please enter Y variable name:")]
+        x_lab = input("Please enter the label for the X-Axis:")
+        y_lab = input("Please enter the label for the Y-Axis:")
+        title = input("Please enter the title of your graph:")
+        return boxplot(x,y,x_lab,y_lab,title);
+    
+    if Q1 == '4':
+        return model_metrics(X_train, X_test, y_train, y_test);
+    
+    if Q1 == '5':
+        Q2= input("What do you want to title the graph?"+"\n")
+        Q3 = input("What is the predictor(dependent) variable name?:"+"\n")
+        Q4 = input("Name of x axis:")
+        Q5 = input("Name of y axis:") 
+        Q6 = input("Choose color for actual y points:"+"\n")
+        Q7 = input("Choose color for line:"+"\n")
+        model_metrics(X_train, X_test, y_train, y_test)
+        return plot_model(X_train, X_test, y_train, y_test,Q2,Q3,Q4,Q5,Q6,Q7)
+
+##### MAIN #####
+
+##Note these four lines of code are meant to test this file 
+#data = pd.read_csv('data.csv')
+#sub_data = data.iloc[1:10,1:10]
+#X = sub_data.iloc[:,0:3]
+#Y = sub_data['readmission_30d']
+
+Xtrain, Xtest, ytrain, ytest = train_test_split(X,Y, test_size = .5) #This assumes that Issue #11 Test.py module works and outputs the test ratio. 
+Xtrain = np.array(Xtrain)
+ytrain = np.array(ytrain)
+Xtest = np.array(Xtest)
+ytest = np.array(ytest)
+
+Q1= input("What type of plot output? (Enter #)"+"\n"+"\n"+"1.Countplot"+"\n"+"2.Scatterplot"+"\n"+"3.Boxplot"+"\n"+"4.Simple Linear Regression Model"+"\n"+"5.Multiple Linear Regression Model"+"\n")
```

Comment:
This line in particular makes this all more like a script than a module.  The mere import of a module generally should not trigger a requirement for user interaction, which this line would do.  You should instead have such user interaction get triggered by calling a function from your module.